app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/Helpers.java

```diff
@@ -1,4 +1,131 @@
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.PositionAssertions.isAbove;
+import static android.support.test.espresso.assertion.PositionAssertions.isBelow;
+import static android.support.test.espresso.assertion.PositionAssertions.isLeftOf;
+import static android.support.test.espresso.assertion.PositionAssertions.isRightOf;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isClickable;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+
 public class Helpers {
+
+    public static boolean checkIfUIObjectIsVisible(Matcher<View> matcher) {
+
+        try {
+            onView(matcher).check(matches(isDisplayed()));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean checkIfUIObjectIsClickable(Matcher<View> matcher) {
+
+        try {
+            onView(matcher).check(matches(isClickable()));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectAbove(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isAbove(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectBelow(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isBelow(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isRemindLeft(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isLeftOf(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isRemindRight(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isRightOf(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static void addItemUppercase(Matcher<View> matcher, String args) {
```

Comment:
The action is "type", not "add". 
Add a better name for "args"

Reply:
Done.